gradio/components.py

```diff
@@ -2723,25 +2722,13 @@ def __init__(
         Parameters:
         value (Any): the initial value of the state.
         """
-        self.value = orjson.loads(value) if isinstance(value, str) else value
-        try:
-            orjson.dumps(self.value, option=orjson.OPT_SERIALIZE_NUMPY)
-        except TypeError:
-            raise ValueError(
-                f"Cannot set initial value because type: {type(value)} is "
-                "not JSON serializable. Instead, you can set the initial "
-                "value to be None, and in your Python function, change it "
-                "to be the your desired value when the function is "
-                "run initially."
-            )
+        self.value = deepcopy(value)
```

Comment:
Might be better to dump before load

```python
orjson.loads(orjson.dumps(value))
```

![image](https://user-images.githubusercontent.com/41651716/176508296-971e3b9d-30d3-4f0c-a19f-64f1ef5bd572.png)

Reply:
That way we don't break the numpy serialization merged before.

Reply:
I got rid of get_config so the value of Variable is never passed to the frontend, so I don't think this is necessary correct?